test/integ_tests/conftest.py

```diff
@@ -161,16 +161,22 @@ def job_failed_name(request):
 
 
 @pytest.fixture(scope="session", autouse=True)
-def completed_quantum_job(aws_session, job_completed_name):
-    account = boto3.client("sts").get_caller_identity().get("Account")
-    region = aws_session.region
-    job = AwsQuantumJob(arn=f"arn:aws:braket:{region}:{account}:job/{job_completed_name}")
-    return job
+def completed_quantum_job(job_completed_name):
+    job_arn = [
+        job["jobArn"]
+        for job in boto3.client("braket").search_jobs(filters=[])["jobs"]
```

Comment:
This can get long as more jobs are run; should we filter creation time to start, say, 1 hour ago?
```suggestion
        for job in boto3.client("braket").search_jobs(
            filters=[
                {
                    "name": "createdAt",
                    "operator": "GT",
                    "values": [(datetime.now(timezone.utc) - timedelta(hours=1)).isoformat()],
                }
            ]
        )["jobs"]
```

Reply:
I guess size isn't the real concern because we're getting only 10 results per page; there _is_ the possibility of the job not being found due to concurrent integ test runs, but that's an extreme edge case.